frb_codegen/src/library/integration/integrator.rs

```diff
@@ -33,33 +46,78 @@ pub fn integrate(config: IntegrateConfig) -> Result<()> {
         .clone()
         .unwrap_or(format!("rust_lib_{}", dart_package_name));
 
+    info!("Extract and modify the Dart project directory");
```

Comment:
for @fzyzcjy (you dont need to do anything): review this after the comment above is changed, since this one depends on that one